generators/client/templates/angular/src/main/webapp/app/shared/_shared-common.module.ts

```diff
@@ -20,10 +20,10 @@ import {
     MaxbytesValidator,
     MinbytesValidator,
     ShowValidationDirective,
-    JhiItemCountComponent,
+    <%=jhiPrefixCapitalized%>ItemCountComponent,
```

Comment:
These shoudnt be changed this should be `Jhi` always as the directive will always be `jhi-*`